esmvaltool/cmorizers/data/formatters/datasets/gcp.py

```diff
@@ -101,7 +100,7 @@ def _extract_variable(short_name, var, cfg, data_table, out_dir):
                         unlimited_dimensions=['time'])
 
 
-def cmorization(in_dir, out_dir, cfg, _):
+def cmorization(in_dir, out_dir, cfg, _, __, ___):
```

Comment:
This is currently not working. It seems that an additional package is needed:
`ImportError: Missing optional dependency 'openpyxl'.  Use pip or conda to install openpyxl.`

Reply:
This added now to the prerequisities